python/lsst/ctrl/bps/wms/panda/conf_example/pipelines_check_idf.yaml

```diff
@@ -2,15 +2,12 @@ pipelineYaml: "${OBS_SUBARU_DIR}/pipelines/DRP.yaml#processCcd"
 
 payload:
   payloadName: pipelines_check
-  runInit: true
-  output: "u/{operator}/{payload_name}"
-  outCollection: "{output}/{timestamp}"
   butlerConfig: s3://butler-us-central1-panda-dev/hsc/butler.yaml
   inCollection: HSC/calib,HSC/raw/all,refcats
   dataQuery: "tract = 9615 and patch=30 and detector IN (10..11) and instrument='HSC' and skymap='hsc_rings_v1' and band in ('r')"
```

Comment:
If really wanted a minimal example, could include the pipelines_check.yaml and only override the butlerConfig (if it doesn't go into a shared yaml).